src/main/java/models/cell/ChargeableCell.java

```diff
@@ -0,0 +1,71 @@
+package models.cell;
+
+
+public abstract  class ChargeableCell implements IChargeable {
+
+    private int id;
+    private String name;
+    private  String type;
```

Comment:
@ashudev6 These properties are already present inside Cell. Extend it. No need to repeat.